confgenerator/files.go

```diff
@@ -35,6 +35,10 @@ func ReadUnifiedConfigFromFile(path, platform string) (UnifiedConfig, error) {
 	if err != nil {
 		return uc, err
 	}
+	if err = uc.Validate(platform); err != nil {
```

Comment:
Doesn't change anything functionally, but conventionally statements like this will have an `err` just for this scope.
```suggestion
	if err := uc.Validate(platform); err != nil {
```